manipulation/src/functTest/java/org/jboss/gm/manipulation/ManifestVerificationFunctionalTest.java

```diff
@@ -186,14 +189,20 @@ public void verifyMongoManifest() throws IOException, URISyntaxException {
         final BuildResult buildResult = TestUtils.createGradleRunner()
                 .withProjectDir(projectRoot)
                 //.withDebug(true)
-                .withArguments("publishToMavenLocal", "--info")
+                .withArguments("assemble", "publish", "--info")
                 .withPluginClasspath()
                 .forwardOutput()
                 .build();
 
-        assertThat(Objects.requireNonNull(buildResult.task(":bson:publishToMavenLocal")).getOutcome())
+        assertThat(Objects.requireNonNull(buildResult.task(":bson:publish")).getOutcome())
                 .isEqualTo(TaskOutcome.SUCCESS);
 
+        Path pathToArtifacts = publishDirectory.toPath().resolve(PATH_IN_REPOSITORY);
+        assertTrue(systemOutRule.getLog().contains(
+                "Updating publication artifactId (driver-core) as it is not consistent with archivesBaseName (mongodb-driver-core)"));
+        assertThat(pathToArtifacts.resolve(ARTIFACT_NAME + ".pom").toFile().exists()).isTrue();
+        assertThat(pathToArtifacts.resolve(ARTIFACT_NAME + ".jar").toFile().exists()).isTrue();
```

Comment:
Can't this just be `assertThat(pathToArtifacts.resolve(ARTIFACT_NAME + ".jar").toFile()).exists();`

Reply:
Huh, probably. For some reason a lot are probably like that :-(

Reply:
Actually, no. Those assertThat are returning an instance of  `BooleanAssert` so it needs an final test (i.e. the `isTrue`, or elesewhere `isFalse` ) otherwise the statement becomes a NOP.